lib/source_file.cpp

```diff
@@ -311,3 +311,7 @@ cstring SourceFileLine::toString() const {
 }
 
 }  // namespace Util
+
+////////////////////////////////////////////////////////
+
+[[gnu::used]] void dbprint(const IHasDbPrint *o) { o->dbprint(std::cout); }
```

Comment:
Could you please add a short comment that the `gnu::used` is required to make sure linker does not omit it?